Dockerfile

```diff
@@ -1,11 +1,16 @@
-FROM docker.io/centos:centos7
+FROM docker.io/centos:centos8
 
-RUN yum install -y python-requests && \
-    curl https://raw.githubusercontent.com/openstack/tripleo-repos/master/tripleo_repos/main.py | python - -b train current-tripleo && \
-    yum update -y && \
-    yum install -y openstack-ironic-python-agent lshw smartmontools iproute python-hardware mdadm biosdevname ipmitool && \
-    yum clean all
+RUN dnf install -y python3 python3-requests && \
+    curl https://raw.githubusercontent.com/openstack/tripleo-repos/master/tripleo_repos/main.py | python3 - -b train current && \
```

Comment:
Note that current isn't the same as current-tripleo - we're now consuming packages which haven't passed tripleo-ci, and previously that has provided some confidence that Ironic is likely to be working - IMO we should consider switching back to current-tripleo when that is available.